iocore/cache/CacheRead.cc

```diff
@@ -585,10 +585,17 @@ CacheVC::openReadReadDone(int event, Event *e)
     }
     // fall through for truncated documents
   }
-Lerror:
+Lerror : {
```

Comment:
because I use local variables and initialization, must use { and }, otherwise occurs compiler error